### DISCUSSION TITLE
It is impossible to determine the license of any code snippets added before the transition to Yari

### DISCUSSION THREAD

**Discussion by danny-laa** [mdn/mdn-community#499] [Content] [answered]
### MDN URL

https://developer.mozilla.org/en-US/docs/MDN/About#copyrights_and_licenses

### What information was incorrect, unhelpful, or incomplete?

It is impossible to determine the license of any code snippets added before the transition to Yari.

### What did you expect to see?

A clear definition of the how is a given code snippets licensed.

### Do you have any supporting links, references, or citations?

The issue has been raised in mdn/content#5901, but **receives no official response** since Feb 2021.

### Do you have anything more you want to share?

According to [MDN](https://developer.mozilla.org/en-US/docs/MDN/About#Copyrights_and_licenses), 

> Code samples added on or after August 20, 2010 are in the public domain (CC0). No licensing notice is necessary, but if you need one, you can use: "Any copyright is dedicated to the Public Domain. http://creativecommons.org/publicdomain/zero/1.0/".
>
> Code samples added to this wiki before August 20, 2010 are available under the MIT license; you should insert the following attribution information into the MIT template: "© <date of last wiki page revision> <name of person who put it in the wiki>".
>
> You can determine which license a code sample is available under by browsing through the article's history. To view the history:
>
> 1. Click the Edit in wiki button in the article header.  This takes you to the same article on the editable, wiki version of MDN Web Docs (but does not actually put the article in edit mode).  
> 2. Click the gear-shaped icon in the header of the wiki article, and choose History in the menu that appears.
> 3. Click View All, and then look for the most recent revision that occurred before August 20, 2010 (if any).
> 4. Click the date stamp of that revision to see the version of the article on that date.

This repo doesn't seem to contain history older than [15 Sep 2020](https://github.com/mdn/content/commits/main?after=3aabc7c9dbe23f846abfc4b63d85b1a4ace796df+11000&branch=main&qualified_name=refs%2Fheads%2Fmain), making it impossible to determine the license of any code snippets added before the date.

**Comment by Josh-Cena** [comment]
@bsmth Maybe this belongs in the community discussions?

This question has been around for quite a while (https://github.com/mdn/content/discussions/5901) but no one is really answering.

**Comment by bsmth** [comment, marked as answer]
Thanks for the ping

>Maybe this belongs in the community discussions

To get to the bottom of it, I think the [existing discussion](https://github.com/mdn/content/discussions/5901) already has some good context. I'll bring it up with the team to see if there's a good way to proceed with this.
